### PR TITLE
Bump version to v0.23.0

### PR DESCRIPTION
**Pull request by MeowZheng**
As title

### CODE REVIEW

File: docs/en/changelog.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+- Support K-Net: Towards Unified Image Segmentation
+- Add `avg_non_ignore` of CELoss to support average loss over non-ignored elements
+- Support dataset initialization with file client
+
+**New Features**
+
+- Support BEiT: BERT Pre-Training of Image Transformers ([1404](https://github.com/open-mmlab/mmsegmentation/pull/1404))
+- Support K-Net: Towards Unified Image Segmentation ([1289](https://github.com/open-mmlab/mmsegmentation/pull/1289))
+- Support dataset initialization with file client ([1402](https://github.com/open-mmlab/mmsegmentation/pull/1402))
+- Add class name function support for STARE datasets ([1376](https://github.com/open-mmlab/mmsegmentation/pull/1376))
```
Comment (Junjun2016) [inline docs/en/changelog.md:17]:
```suggestion
- Add class name function for STARE datasets ([1376](https://github.com/open-mmlab/mmsegmentation/pull/1376))
```

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+- Support different seeds to different ranks when distributed training ([1362](https://github.com/open-mmlab/mmsegmentation/pull/1362))
```
Comment (Junjun2016) [inline docs/en/changelog.md:18]:
```suggestion
- Support different seeds on different ranks when distributed training ([1362](https://github.com/open-mmlab/mmsegmentation/pull/1362))
```

**Review by Junjun2016 — APPROVED** [review]

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,5 +1,51 @@
...
+- Add nlc2nchw2nlc and nchw2nlc2nchw ([1249](https://github.com/open-mmlab/mmsegmentation/pull/1249))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:19]:
```suggestion
- Add `nlc2nchw2nlc` and `nchw2nlc2nchw` to reshape tensor easier ([1249](https://github.com/open-mmlab/mmsegmentation/pull/1249))
```

File: docs/en/changelog.md [hunk 4]
```diff
@@ -1,5 +1,51 @@
...
+
+**Improvements**
+
+- Synchronize random seed for distributed sampler ([#1411](https://github.com/open-mmlab/mmsegmentation/pull/1411))
+- Add faq document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:24]:
```suggestion
- Add FAQ document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
```
FAQ is abbreviation for "frequently asked questions", which should use capital letter.

**Review by Junjun2016 — APPROVED** [review]

File: docs/en/changelog.md [hunk 5]
```diff
@@ -1,5 +1,51 @@
...
+- Add script and documentation for multi-machine distributed training ([1383](https://github.com/open-mmlab/mmsegmentation/pull/1383))
+
+**Bug Fixes**
+
+- Add `avg_non_ignore` of CELoss to support average loss over non-ignored elements ([1409](https://github.com/open-mmlab/mmsegmentation/pull/1409))
+- Fix some wrong urls of models or log in ./configs ([#1336](https://github.com/open-mmlab/mmsegmentation/pull/1433))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:30]:
```suggestion
- Fix some wrong URLs of models or logs in `./configs` ([#1336](https://github.com/open-mmlab/mmsegmentation/pull/1433))
```

File: docs/en/changelog.md [hunk 6]
```diff
@@ -1,5 +1,51 @@
...
+- Make arguments effective in tools/confusion_matrix.py ([1401](https://github.com/open-mmlab/mmsegmentation/pull/1401))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:30]:
```suggestion
- Add title and color theme arguments to plot function: `./tools/confusion_matrix.py` ([1401](https://github.com/open-mmlab/mmsegmentation/pull/1401))
```

File: docs/en/changelog.md [hunk 7]
```diff
@@ -1,5 +1,51 @@
...
+- Fix outdated link in Colab demo ([1392](https://github.com/open-mmlab/mmsegmentation/pull/1392))
+- Fix typos ([1424](https://github.com/open-mmlab/mmsegmentation/pull/1424) [1405](https://github.com/open-mmlab/mmsegmentation/pull/1405), [1371](https://github.com/open-mmlab/mmsegmentation/pull/1371) [1366](https://github.com/open-mmlab/mmsegmentation/pull/1366))
+
+**Documentation**
+
+- Add faq document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
```
Comment (MengzhangLI) [inline docs/en/changelog.md]:
```suggestion
- Add FAQ document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
```

File: docs/en/changelog.md [hunk 8]
```diff
@@ -1,5 +1,51 @@
...
+- Fix the config name style description ([1414](https://github.com/open-mmlab/mmsegmentation/pull/1414))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:37]:
```suggestion
- Fix the config name style description in official docs ([1414](https://github.com/open-mmlab/mmsegmentation/pull/1414))
```

File: docs/en/changelog.md [hunk 9]
```diff
@@ -1,5 +1,51 @@
...
+- Update config.md ([1363](https://github.com/open-mmlab/mmsegmentation/pull/1363))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:38]:
```suggestion

```

File: docs/en/changelog.md [hunk 10]
```diff
@@ -1,5 +1,51 @@
...
+- Add faq document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
+- Add script and documentation for multi-machine distributed training ([1383](https://github.com/open-mmlab/mmsegmentation/pull/1383))
+
+**Bug Fixes**
+
+- Add `avg_non_ignore` of CELoss to support average loss over non-ignored elements ([1409](https://github.com/open-mmlab/mmsegmentation/pull/1409))
+- Fix some wrong urls of models or log in ./configs ([#1336](https://github.com/open-mmlab/mmsegmentation/pull/1433))
+- Make arguments effective in tools/confusion_matrix.py ([1401](https://github.com/open-mmlab/mmsegmentation/pull/1401))
+- Fix outdated link in Colab demo ([1392](https://github.com/open-mmlab/mmsegmentation/pull/1392))
+- Fix typos ([1424](https://github.com/open-mmlab/mmsegmentation/pull/1424) [1405](https://github.com/open-mmlab/mmsegmentation/pull/1405), [1371](https://github.com/open-mmlab/mmsegmentation/pull/1371) [1366](https://github.com/open-mmlab/mmsegmentation/pull/1366))
```
Comment (MengzhangLI) [inline docs/en/changelog.md:32]:
```suggestion
- Fix typos ([1424](https://github.com/open-mmlab/mmsegmentation/pull/1424) [1405](https://github.com/open-mmlab/mmsegmentation/pull/1405), [1371](https://github.com/open-mmlab/mmsegmentation/pull/1371) [1366](https://github.com/open-mmlab/mmsegmentation/pull/1366) [1363](https://github.com/open-mmlab/mmsegmentation/pull/1363))
```

**Review by MengzhangLI — APPROVED** [review]

**Pull request merged**